tools/clang/unittests/HLSL/DxilContainerTest.cpp

```diff
@@ -204,7 +204,13 @@ class DxilContainerTest : public ::testing::Test {
     std::string baseName(baseDesc.Name);
     if (baseName.compare(0, 4, "half", 4) == 0)
       baseName = baseName.replace(0, 4, "float", 5);
-    VERIFY_ARE_EQUAL_STR(testDesc.Name, baseName.c_str());
+
+    // For anonymous structures, fxc uses "scope::<unnamed>",
+    // dxc uses "anon", and may have additional ".#" for name disambiguation.
+    // Just skip name check if struct is unnamed according to fxc.
+    if (baseName.find("<unnamed>") == std::string::npos) {
```

Comment:
I wish it was as easy to preserve scope and match fxc here, but clang doesn't preserve any scope for anonymous structures in CodeGen.